target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -122,6 +122,8 @@ BTHOMEHUBV2B|BTHOMEHUBV3A)
 	lan_mac=$(mtd_get_mac_ascii uboot_env ethaddr)
 	wan_mac=$(macaddr_add "$lan_mac" 1)
 	ucidef_set_interface_lan 'eth0'
```

Comment:
Ah, that would have been easier. Yes, I now about the downsides of force pushing for anything cloned by others 👍